.ado/publish.yml

```diff
@@ -0,0 +1,48 @@
+# Publishing VSCode Extensions https://code.visualstudio.com/api/working-with-extensions/publishing-extension
+#   * $(PAT): https://stackoverflow.com/questions/58192624/publishing-vs-code-extension-via-azure-devops/58195815#58195815
+#   * Extension Manifest (package.json): https://code.visualstudio.com/api/references/extension-manifest
+
+variables:
+  Ver.Major: 0
+  Ver.Minor: 0
+  Ver.Patch: 0
+
+name: $(BuildId)
+
+trigger: none
+
+jobs:
+- job: "Publish"
+  pool: 
+    vmImage: 'windows-latest'
```

Comment:
It'll probably build quicker on Linux, unless there's a reason to use the Windows pool.

Reply:
I'll try.

Reply:
Wow! In ubuntu it is about 2 times faster. Thanks! :-D